NuRadioMC/simulation/simulation.py

```diff
@@ -521,7 +521,7 @@ def run(self):
                             from matplotlib import pyplot as plt
                             fig, (ax, ax2) = plt.subplots(1, 2)
                             ax.plot(self._ff, np.abs(eTheta) / units.micro / units.V * units.m)
-                            ax2.plot(self._tt, fft.freq2time(eTheta) / units.micro / units.V * units.m)
+                            ax2.plot(self._tt, fft.freq2time(eTheta, self._dt) / units.micro / units.V * units.m)
```

Comment:
again here, shouldn't it be `1/self._dt`?

Reply:
Yes, you are right. I fixed it